psalm-baseline.xml

```diff
@@ -97,6 +97,9 @@
     <UndefinedClass>
       <code><![CDATA[LightnCandy]]></code>
     </UndefinedClass>
+    <UnusedMethod>
+      <code><![CDATA[approversToViewModel]]></code>
+    </UnusedMethod>
```

Comment:
I'm very confused why Psalm complains about this method being unused: https://github.com/ProfessionalWiki/PageApprovals/actions/runs/10269500301

This PR doesn't touch it and the code still

Reply:
Looks like we already have other unexpected exclusions from the same file: https://github.com/ProfessionalWiki/PageApprovals/blob/69bbcebaf09a57a5727533a32623db19d36a48f8/psalm-baseline.xml#L100-L102